test/WebJobs.Script.Tests/WebJobsScriptHostServiceTests.cs

```diff
@@ -87,6 +88,33 @@ private Mock<IHost> CreateMockHost(SemaphoreSlim disposedSemaphore = null)
             return host;
         }
 
+        [Fact]
+        public async Task StartAsync_Succeeds()
+        {
+            var hostBuilder = new Mock<IScriptHostBuilder>();
+            hostBuilder.Setup(b => b.BuildHost(It.IsAny<bool>(), It.IsAny<bool>())).Returns(_host.Object);
+
+            _webHostLoggerProvider = new TestLoggerProvider();
+            _loggerFactory = new LoggerFactory();
+            _loggerFactory.AddProvider(_webHostLoggerProvider);
+
+            var mockEventManager = new Mock<IScriptEventManager>(MockBehavior.Strict);
+            mockEventManager.Setup(p => p.Publish(It.IsAny<HostStartEvent>()));
+
+            var metricsLogger = new TestMetricsLogger();
+            _hostService = new WebJobsScriptHostService(
+                _monitor, hostBuilder.Object, _loggerFactory,
+                _mockScriptWebHostEnvironment.Object, _mockEnvironment.Object,
+                _hostPerformanceManager, _healthMonitorOptions,
+                metricsLogger, new Mock<IApplicationLifetime>().Object,
+                _mockConfig, mockEventManager.Object);
+
+            await _hostService.StartAsync(CancellationToken.None);
+
+            // add general post startup validations here
+            mockEventManager.Verify(_ => _.Publish(It.IsAny<HostStartEvent>()), Times.Once());
```

Comment:
Added this StartAsync test so I had a place to add verification for the new event